packages/cosmo/package.py

```diff
@@ -205,11 +204,18 @@ def test(self):
                     env['REAL_TYPE'] = 'FLOAT'
                 if '~cppdycore' in self.spec:
                     env['JENKINS_NO_DYCORE'] = 'ON'
-                run_testsuite = Executable('sbatch submit.' + self.spec.variants['slave'].value + '.slurm')
-                run_testsuite()
+                run_testsuite = 'sbatch -W submit.' + self.spec.variants['slave'].value + '.slurm'
+                os.system(run_testsuite)
+                cat_testsuite = 'cat testsuite.out'
```

Comment:
I have the same comment I made in the other PR, I would do a more pythonic
```output=subprocess.run('sbatch ...', capture_output=True)
print(output.stdout)
test = subprocess.run('./testfail.sh')
if test.returncode:
   raise... 
```
but also again I would do that in python3, so we can merge this and postpone the change